[2021-05-09 19:25:32,620] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.python_push_into_target 2021-05-07T00:00:00+00:00 [queued]>
[2021-05-09 19:25:32,652] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.python_push_into_target 2021-05-07T00:00:00+00:00 [queued]>
[2021-05-09 19:25:32,654] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 19:25:32,656] {taskinstance.py:1069} INFO - Starting attempt 11 of 13
[2021-05-09 19:25:32,657] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 19:25:32,826] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): python_push_into_target> on 2021-05-07T00:00:00+00:00
[2021-05-09 19:25:32,840] {standard_task_runner.py:52} INFO - Started process 21992 to run task
[2021-05-09 19:25:32,880] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_4', 'python_push_into_target', '2021-05-07T00:00:00+00:00', '--job-id', '149', '--pool', 'default_pool', '--raw', '--subdir', '/home/navneetsajwan/airflow/dags/dag__4.py', '--cfg-path', '/tmp/tmp2xgz3wbv', '--error-file', '/tmp/tmpsswdw9ad']
[2021-05-09 19:25:32,884] {standard_task_runner.py:77} INFO - Job 149: Subtask python_push_into_target
[2021-05-09 19:25:33,137] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_4.python_push_into_target 2021-05-07T00:00:00+00:00 [running]> on host navneetsajwan-Lenovo-G50-80
[2021-05-09 19:25:33,241] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=navneet@metaorigins.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag_4
AIRFLOW_CTX_TASK_ID=python_push_into_target
AIRFLOW_CTX_EXECUTION_DATE=2021-05-07T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-07T00:00:00+00:00
[2021-05-09 19:25:35,360] {logging_mixin.py:104} INFO - push_test
[2021-05-09 19:25:35,360] {python.py:118} INFO - Done. Returned value was: None
[2021-05-09 19:25:35,368] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=dag_4, task_id=python_push_into_target, execution_date=20210507T000000, start_date=20210509T135532, end_date=20210509T135535
[2021-05-09 19:25:35,578] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-09 19:25:35,593] {local_task_job.py:146} INFO - Task exited with return code 0
